src/jitlayers.cpp

```diff
@@ -244,6 +244,10 @@ void addOptimizationPasses(legacy::PassManagerBase *PM, int opt_level)
 #endif
 }
 
+void jl_add_optimization_passes(LLVMPassManagerRef PM, int opt_level) {
```

Comment:
if you have DLLEXPORT on the prototype, it also needs to be on the implementation